{"uuid": "d799716a-575b-45f7-807f-f175c639907c", "children": ["ac02fc18-8004-41e5-b3d5-bd5924718761"], "befores": [{"name": "init_environment", "status": "failed", "statusDetails": {"message": "AssertionError: assert 401 == 205\n +  where 401 = <Response [401]>.status_code\n", "trace": "  File \"/usr/local/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/usr/local/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/usr/local/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1132, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.10/site-packages/_pytest/fixtures.py\", line 913, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/usr/workspace/conftest.py\", line 18, in init_environment\n    assert response.status_code == 205\n"}, "start": 1714031094179, "stop": 1714031096478}], "start": 1714031094179, "stop": 1714031096619}